### PR TITLE
Create CSVReader

### PR DESCRIPTION
**Pull request by shadetree01010100**

### CODE REVIEW

File: tests/test_csv_reader_block.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+from nio.signal.base import Signal
+from nio.testing.block_test_case import NIOBlockTestCase
+from unittest.mock import patch, MagicMock, mock_open
+
+from ..csv_reader_block import CSVReader
+
+
+class TestReadLines(NIOBlockTestCase):
```
Comment (tyoungNIO) [inline tests/test_csv_reader_block.py]: todo: test `loop` behavior

**Comment by cowleyk** [comment]
The `release.json` will also need to be updated to include this block

File: spec.json [hunk 2]
```diff
@@ -1,7 +1,37 @@
 {
+  "nio/CSVReader": {
+    "version": "0.1.0",
+    "description": "Read lines from a CSV file. For each incoming signal, one row is read from `file`.",
+    "categories": [],
```
Comment (cowleyk) [inline spec.json]: needs a category assigned

Comment (cowleyk) [reply spec.json]: `File System`

**Review by cowleyk — APPROVED** [review]

**Pull request merged**